dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -0,0 +1,2118 @@
+            }
+        }
+
+
+        list = itemService.getMetadata(publication, "dc", "contributor", "author", Item.ANY);
+
+        for (MetadataValue mdv : list) {
+            if (StringUtils.equals(mdv.getValue(), "plain text three")) {
+                assertEquals(4, mdv.getPlace());
+            }
+        }
+
+
+        getClient(adminToken).perform(get("/api/core/relationships/" + thirdRelationshipIdString))
+                             .andExpect(status().isOk())
+                             .andExpect(jsonPath("leftPlace", is(3)));
+
+    }
+
+    /**
+     * This test will simply add Relationships between Items with a useForPlace attribute set to false for the
+     * RelationshipType. We want to test that the Relationships that are created will still have their place
+     * attributes handled in a correct way
+     * @throws Exception
+     */
+    @Test
+    public void addRelationshipsNotUseForPlace() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+        Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
+        Collection col3 = CollectionBuilder.createCollection(context, child1).withName("OrgUnits").build();
+
+        Item author1 = ItemBuilder.createItem(context, col1)
+                                  .withTitle("Author1")
+                                  .withIssueDate("2017-10-17")
+                                  .withAuthor("Smith, Donald")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item author2 = ItemBuilder.createItem(context, col2)
+                                  .withTitle("Author2")
+                                  .withIssueDate("2016-02-13")
+                                  .withAuthor("Smith, Maria")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item author3 = ItemBuilder.createItem(context, col2)
+                                  .withTitle("Author3")
+                                  .withIssueDate("2016-02-13")
+                                  .withAuthor("Maybe, Maybe")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item orgUnit1 = ItemBuilder.createItem(context, col3)
+                                   .withTitle("OrgUnit1")
+                                
```

Comment:
as above

Reply:
@abollini : again, this comment is vague (and oddly, GitHub put it on an empty line).  I'm not sure what you are requesting here. Could you clarify?

Reply:
it is strange. If I look to the conversation tab I see it in the wrong place but it looks fine now in the files tab (so yes, it is a github bug).
The comment is about
https://github.com/DSpace/DSpace/pull/2376/files#diff-61a3266941bcb44b91094e4d3f330ba6R1316

```
content = mvcResult.getResponse().getContentAsString();
        map = mapper.readValue(content, Map.class);
        String secondRelationshipIdString = String.valueOf(map.get("id"));
```

and I'm point to my previous comment here https://github.com/DSpace/DSpace/pull/2376/files#r279211896